### PR TITLE
Task-37845 : Active DLP document preview

### PR DESCRIPTION
**Pull request by ahmadmido**

### CODE REVIEW

**Review by rdenarie — DISMISSED** [review]

File: core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java [hunk 1]
```diff
@@ -227,5 +169,21 @@ public void removePositiveItem(String itemReference) {
...
+
+  @Override
+  public String getItemUrl(String itemReference) {
+    ExtendedSession session = null;
+    try {
+      session =
+          (ExtendedSession) WCMCoreUtils.getSystemSessionProvider()
+                                        .getSession(COLLABORATION_WS, repositoryService.getCurrentRepository());
+      Node node = session.getNodeByIdentifier(itemReference);
+      return WCMCoreUtils.getLinkInDocumentsApplication(node.getPath());
```
Comment (rdenarie) [inline core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java:187]: can show me the path generated by this  ?
Which drive is used ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java (hunk 1 above)
Comment (ahmadmido) [reply core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java:187]: This is an example of a path generated: /portal/dw/documents?path=Security%2FSecurity%2Flegume%252520205008770.xlsx

Comment (rdenarie) [reply core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java:187]: Ok for me

**Review by rdenarie — DISMISSED** [review]

**Review by rdenarie — APPROVED** [review]

File: core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java [hunk 2]
```diff
@@ -176,6 +178,21 @@ public void removePositiveItem(String itemReference) {
...
+    } catch (Exception e) {
+      LOGGER.error("Error while deleting dlp file item", e);
```
Comment (rdenarie) [inline core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java]: Can you update the log ? it is not deletion :)

**Review by rdenarie — CHANGES_REQUESTED** [review]

**Review by rdenarie — DISMISSED** [review]

**Review by rdenarie — APPROVED** [review]

**Pull request merged**